### PR TITLE
Increase HW requirements for RPC nodes

### PR DESCRIPTION
**Pull request by telezhnaya**
All the costs are taken for 128 GB memory instance, I decided it's a good median value

### CODE REVIEW

**Comment by VanBarbascu** [comment]
My understanding is that nodes tracking one shard can work with 32GB. I believe that the min requirement at 80 GB is a bit too high.

**Comment by telezhnaya** [comment]
> nodes tracking one shard

RPC nodes should track all the shards

**Comment by VanBarbascu** [comment]
Yes, but not in memory.

File: docs/archival/hardware-archival.md [hunk 1]
```diff
@@ -8,21 +8,52 @@ description: NEAR Archival Node Hardware Requirements
...
+# Testnet
+
+## Recommended Hardware Specifications {#recommended-hardware-specifications-testnet}
+
+| Hardware       | Recommended Specifications                                      |
+| -------------- |-----------------------------------------------------------------|
+| CPU            | 8-Core (16-Thread) Intel i7/Xeon or equivalent with AVX support |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI                 |
+| RAM            | 24GB DDR4                                                       |
+| Hot Storage    | 3 Terabyte SSD                                                  |
```
Comment (andrei-near) [inline docs/archival/hardware-archival.md:53]: Testnet RPC DB for 5 epochs if ~540GB. 
1 TB disk should be more than enough.

File: docs/archival/hardware-archival.md [hunk 2]
```diff
@@ -8,21 +8,52 @@ description: NEAR Archival Node Hardware Requirements
...
+| Cold Storage   | 40 Terabyte  NON-SSD Persistent Disks                           |
```
Comment (andrei-near) [inline docs/archival/hardware-archival.md:54]: Cold DB is currently 11 TB.
With spare, 14-15TB should be enough

File: docs/archival/hardware-archival.md [hunk 3]
```diff
@@ -31,25 +62,23 @@ _Verify AVX support on Linux by issuing the command ```$ lscpu | grep -oh  avx``
 | Hot Storage    | 3 Terabyte SSD                                                             |
```
Comment (andrei-near) [inline docs/archival/hardware-archival.md:62]: ditto

File: docs/archival/hardware-archival.md [hunk 4]
```diff
@@ -31,25 +62,23 @@ _Verify AVX support on Linux by issuing the command ```$ lscpu | grep -oh  avx``
...
 | Cold Storage   | 40 Terabyte NON-SSD Persistent Disks                                       |
```
Comment (andrei-near) [inline docs/archival/hardware-archival.md:63]: ditto

File: docs/archival/hardware-archival.md [hunk 5]
```diff
@@ -31,25 +62,23 @@ _Verify AVX support on Linux by issuing the command ```$ lscpu | grep -oh  avx``
...
-## Cost Estimation {#cost-estimation}
-
-Estimated monthly costs depending on operating system:
+Estimated monthly costs depending on cloud provider:
 
 | Cloud Provider | Machine Size    | Linux                     |
-| -------------- | --------------- | ------------------------  |
-| AWS            | m5.2xlarge      | $330 CPU + $800 storage † |
+| -------------- | --------------- |---------------------------|
+| AWS            | m5a.2xlarge     | $160 CPU + $800 storage † |
```
Comment (andrei-near) [inline docs/archival/hardware-archival.md:71]: storage costs should be lower since disks are less than half

File: docs/rpc/hardware-rpc.md [hunk 6]
```diff
@@ -6,47 +6,75 @@ sidebar_position: 1
...
-| CPU            | 8-Core (16-Thread) Intel i7/Xeon or equivalent                           |
-| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI                          |
-| RAM            | 20GB DDR4                                                                 |
-| Storage        | 1TB SSD (NVMe SSD is recommended. HDD will be enough for localnet only ) |
+| Hardware       | Recommended Specifications                      |
+| -------------- |-------------------------------------------------|
+| CPU            | 8-Core (16-Thread) Intel i7/Xeon or equivalent  |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI |
+| RAM            | 32GB DDR4                                       |
+| Storage        | 1TB NVMe SSD                                    |
```
Comment (andrei-near) [inline docs/rpc/hardware-rpc.md:22]: 2.4TB disk for RPC

File: docs/rpc/hardware-rpc.md [hunk 7]
```diff
@@ -6,47 +6,75 @@ sidebar_position: 1
...
 
-_Verify AVX support on Linux by issuing the command ```$ lscpu | grep -oh  avx```. If the output is empty, your CPU is not supported._
+## Minimal Hardware Specifications {#minimal-hardware-specifications}
 
+| Hardware       | Minimal Specifications                          |
+| -------------- |-------------------------------------------------|
+| CPU            | 8-Core (16-Thread) Intel i7/Xeon or equivalent  |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI |
+| RAM            | 24GB DDR4                                       |
+| Storage        | 500GB NVMe SSD                                  |
```
Comment (andrei-near) [inline docs/rpc/hardware-rpc.md:31]: 2.4TB disk

File: docs/rpc/hardware-rpc.md [hunk 8]
```diff
@@ -6,47 +6,75 @@ sidebar_position: 1
...
+| Storage        | 1TB NVMe SSD                                    |
+
+## Minimal Hardware Specifications {#minimal-hardware-specifications-testnet}
+
+| Hardware       | Minimal Specifications                          |
+| -------------- |-------------------------------------------------|
+| CPU            | 8-Core (16-Thread) Intel i7/Xeon or equivalent  |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI |
+| RAM            | 24GB DDR4                                       |
+| Storage        | 500GB NVMe SSD                                  |
```
Comment (andrei-near) [inline docs/rpc/hardware-rpc.md:61]: 600GB min (DB size is 540GB)

File: docs/validator/hardware-validator.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+## Recommended Hardware Specifications {#recommended-hardware-specifications}
+
+### Chunk/Block Producers
+
+| Hardware       | Recommended Specifications                                   |
+| -------------- |--------------------------------------------------------------|
+| CPU            | x86_64 (Intel, AMD) processor with at least 8 physical cores |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI              |
+| RAM            | 32GB DDR4                                                    |
+| Storage        | 2TB NVMe SSD                                                 |
```
Comment (andrei-near) [inline docs/validator/hardware-validator.md:74]: 1TB

File: docs/validator/hardware-validator.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+
+### Chunk validators
+
+| Hardware       | Recommended Specifications                                   |
+| -------------- |--------------------------------------------------------------|
+| CPU            | x86_64 (Intel, AMD) processor with at least 8 physical cores |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI              |
+| RAM            | 32GB DDR4                                                    |
+| Storage        | 2TB NVMe SSD                                                 |
```
Comment (andrei-near) [inline docs/validator/hardware-validator.md:83]: 1TB

File: docs/validator/hardware-validator.md [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+## Minimal Hardware Specifications {#minimal-hardware-specifications}
+
+### Chunk/Block Producers
+
+| Hardware       | Minimal Specifications                                       |
+| -------------- |--------------------------------------------------------------|
+| CPU            | x86_64 (Intel, AMD) processor with at least 8 physical cores |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI              |
+| RAM            | 24GB DDR4                                                    |
+| Storage        | 1TB NVMe SSD                                                 |
```
Comment (andrei-near) [inline docs/validator/hardware-validator.md:94]: 600GB minimum

File: docs/validator/hardware-validator.md [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+
+### Chunk validators
+
+| Hardware       | Minimal Specifications                                       |
+| -------------- |--------------------------------------------------------------|
+| CPU            | x86_64 (Intel, AMD) processor with at least 8 physical cores |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI              |
+| RAM            | 24GB DDR4                                                    |
+| Storage        | 1TB NVMe SSD                                                 |
```
Comment (andrei-near) [inline docs/validator/hardware-validator.md:103]: 600GB minimum

File: docs/rpc/hardware-rpc.md (hunk 6 above)
Comment (andrei-near) [reply docs/rpc/hardware-rpc.md:22]: We could also add a Note explaining that rpc/hot storage is based on gc_num_epochs_to_keep = 5.
Decreasing this means less disk required, increasing - more data stored and higher disk usage.

**Review by andrei-near — CHANGES_REQUESTED** [review]

File: docs/rpc/hardware-rpc.md (hunk 6 above)
Comment (telezhnaya) [reply docs/rpc/hardware-rpc.md:22]: Min gc_num_epochs_to_keep is 3, right?
Do you know how much ssd is required with gc_num_epochs_to_keep = 3?

Comment (andrei-near) [reply docs/rpc/hardware-rpc.md]: Don't know about 3 epochs as we run all nodes with 5.
Don't think this is a critical piece of information but rather a baseline for these numbers.

**Review by andrei-near — APPROVED** [review]

**Pull request merged**